Modules/Devices/test/Devices.Application.Tests/Tests/Identities/Queries/ListIdentities/FindAllStubRepository.cs

```diff
@@ -17,4 +18,9 @@ public Task<DbPaginationResult<Identity>> FindAll(PaginationFilter paginationFil
     {
         return Task.FromResult(_identities);
     }
+
+    public Task<Identity> FindByAddress(IdentityAddress address, CancellationToken cancellationToken)
+    {
+        return Task.FromResult(_identities.ItemsOnPage.Where(it=>it.Address==address).First());
```

Comment:
throw a NotImplementedException. This StubRepository should (as its name suggests) only implement the FindAll method